### DISCUSSION TITLE
Award claim blocks upon unlocking advancements

### DISCUSSION THREAD

**Discussion by frisno-b** [GriefPrevention/GriefPrevention#2063] [Ideas]
Award claim blocks upon unlocking advancements/recipes I'd like the options to automatically bestow claim blocks for selected advancements,achievements, recipes in order to award active gameplay in addition to time online.

For example: Player unlocked recipe Glass Panes. They're probably building a house. Lets give them x blocks at this time in need.

**Comment by Jikoo** [comment]
You can add functions to advancements as rewards. The function can then execute a command when the advancement is achieved.
https://minecraft.fandom.com/wiki/Function_(Java_Edition)#Invocation_from_advancements
Alternately, an addon could do this.

**Reply by frisno-b** [reply]
Thank you, I'll check it out.
